src/Alloy.Mvc.Template/Models/Pages/SitePageData.cs

```diff
@@ -11,7 +11,7 @@ namespace AlloyTemplates.Models.Pages
     /// <summary>
     /// Base class for all page types
     /// </summary>
-    public abstract class SitePageData : PageData, ICustomCssInContentArea
```

Comment:
PageData does already implement ILocalizable, please remove that change and I will accept your PR